api/v1alpha1/image_types.go

```diff
@@ -317,6 +319,7 @@ type ImageFilter struct {
 // ImageResourceStatus represents the observed state of a Glance image
 type ImageResourceStatus struct {
 	// status is the image status as reported by Glance
+	// +kubebuilder:validation:MaxLength=1024
```

Comment:
Can you add a validation test for this? I suspect it might not work as the type is *string, not string.

Reply:
...or we could just change the type to `string` 🤔

Lets do that. I don't think we need to distinguish between empty and unset here.

Reply:
I can change the the type to `string` if it makes things easier. Also, I don't think we need a test as we don't really care about what goes into the field, but our intention with this validation is to protect our CEL validation budget.